runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -46,8 +46,8 @@
     protected abstract UUID sendRequest(ManagementClient client) throws TimeoutException;
 
     /**
-     * Select an orchestrator and return a client. Orchestrator's that
-     * are on unresponsive nodes and the affected endpoint by the workflow
+     * Select an orchestrator and return a client. Orchestrator
+     * located on the affected by the workflow endpoint
```

Comment:
Orchestrator located on the affected node and nodes not responding to pings will not be selected.

Reply:
sounds good